manim/scene/graph_scene.py

```diff
@@ -260,25 +258,26 @@ def setup_axes(self, animate=False):
         self.default_graph_colors = it.cycle(self.default_graph_colors)
 
     def coords_to_point(self, x, y):
-        """
-        The graph is smaller than the scene.
-        Because of this, coordinates in the scene don't map
-        to coordinates on the graph.
-        This method returns a scaled coordinate for the graph,
-        given cartesian coordinates that correspond to the scene..
+        """The graph is smaller than the scene.
+
+        Because of this, coordinates in the graph don't map
+        to coordinates on the scene.
+
+        This method returns a scaled coordinate for the scene,
+        given coordinates that correspond to the graph.
```

Comment:
In the way this is currently written, it will make "The graph is smaller than the scene." show up as the one-line documentation of this function. Suggestion for improvement:

```suggestion
        """Maps a pair of coordinates (with respect to the axes)
        to a point in the scene.

        The graph is smaller than the scene. Because of this, coordinates
        in the graph don't map to coordinates on the scene.

        This method returns a scaled coordinate for the scene,
        given coordinates that correspond to the graph.
```